proxy/http2/Http2DependencyTree.h

```diff
@@ -196,6 +197,8 @@ Tree<T>::add(uint32_t parent_id, uint32_t id, uint32_t weight, bool exclusive, T
         node->queue->push(child->entry);
       }
       node->children.push(child);
+      ink_release_assert(child != node);
+      ink_release_assert(!in_parent_chain(child, node));
```

Comment:
What's the performance cost of this?

Reply:
yes, as noted above, I probably want to pull out some of these extra asserts.